### PR TITLE
Update LargeScale.md

### PR DESCRIPTION
**Pull request by leerho**
@ZacBlanco brought to our attention that this document was inadequate in explaining how we support cross-platform and cross-language compatibility.  

This revised document is intended to reflect our cross-platform and cross-language capabilities more accurately.

### CODE REVIEW

File: docs/Architecture/LargeScale.md [hunk 1]
```diff
@@ -21,20 +21,47 @@ layout: doc_page
 -->
 ## Designed for Large-scale Computing Systems
 
+#### Multiple Languages
+
+* The DataSketches library is now available in three languages, Java, C++, and Python. A forth language, GoLang, is in development.
```
Comment (freakyzoidberg) [inline docs/Architecture/LargeScale.md]: `A fourth language, Go, is in development.`

Typo in forth/fourth
GoLang is the former name of Go https://go.dev/doc/faq#go_or_golang

File: docs/Architecture/LargeScale.md [hunk 2]
```diff
@@ -21,20 +21,47 @@ layout: doc_page
...
+
+#### The Serialized Image of a Sketch
+* The structure (or image) of a serialized sketch is independent of the language from which it was created. 
+* The sketch image only contains little-endian primitives, such as int64, int32, int16, int8, double-64, float-32, UTF-8 strings, and simple array structures of those, which can be easily interpreted in many languages on modern CPUs. We do not support big-endian serialization.
+* The sketch image is unique for each type of sketch.
+* Simply speaking, a sketch image can be viewed as a blob of bytes, which is easily stored and easily transported using many different protocols, including Protobuf, Avro, Thrift, Byte64, etc.
+
```
Comment (freakyzoidberg) [inline docs/Architecture/LargeScale.md:51]: Should we clarify that for some sketches (FrequencyItemSketch iirc) the serialised form between language may not be strictly equal but still be logically equivalent?

Comment (leerho) [reply docs/Architecture/LargeScale.md:51]: Excellent point.

File: docs/Architecture/LargeScale.md (hunk 1 above)
Comment (leerho) [reply docs/Architecture/LargeScale.md]: Done.

File: docs/Architecture/LargeScale.md (hunk 2 above)
Comment (leerho) [reply docs/Architecture/LargeScale.md:51]: Fixed. Thank you!

File: docs/Architecture/LargeScale.md [hunk 3]
```diff
@@ -21,20 +21,47 @@ layout: doc_page
...
+From the user's perspective, as long as the *SerVer* is the same, older *Software Versions* should be able to read sketch images created by newer *Software Versions*. But the APIs may be different, obviously. An older *Software Version* will not be able to take advantage of new features introduced in new *Software Versions*, but it should be able to do what it did before. In other words, there will be no loss of access to the serialized sketch and the older *Software Version* capabilities. A user should not need to access the *SerVer*, nonetheless it is always stored in index one of the serialized image. If a sketch is presented with a *SerVer* that it is not compatible with, the sketch should throw an exception and say what the problem is, just like Java does with its *Class File Format Versions*.
+
```
Comment (freakyzoidberg) [inline docs/Architecture/LargeScale.md:43]: Sorry one more comment, for Generic based sketches, using custom SerDe, the onus is on the user to guarantee the compatibility across version of their custom SerDe

I suppose that's very implied, but maybe worth mentioning that we do not have much guarantee here.

Comment (leerho) [reply docs/Architecture/LargeScale.md:43]: Right again. We had a comment wrt to that in an earlier version, but it didn't get in to this one. Will Fix.

**Review by freakyzoidberg — APPROVED** [review]

**Pull request merged**